src/Evented.ts

```diff
@@ -36,6 +36,22 @@ function handlesArraytoHandle(handles: Handle[]): Handle {
 	};
 }
 
+export interface EventObject {
+	/**
+	 * The type of the event
+	 */
+	readonly type: string | symbol;
```

Comment:
Are we going to get into assignability issues with some code taking `EventObject` from `@dojo/interfaces/core` and some coming from this with a `symbol` in there?